R/plot_forecast.R

```diff
@@ -0,0 +1,145 @@
+#' Plot forecasts with truth for only one model, one target and one location
+#'
+#' @param forecast_data data frame with truth and forecasts from load_forecasts()
+#' @param model model_abbr specifying model to plot
+#' @param target_variable string specifying target type. It should be one of 
+#' "Cumulative Cases","Cumulative Deaths","Incident Cases" and "Incident Deaths,"
+#' @param location string for fips code or 'US'
+#' @param intervals values indicating which central prediction interval levels 
+#' to plot, defaults to c(.5, .8, .95). NULL means only plotting point forecasts.
+#' @param horizon forecasts are plotted for the horizon time steps after the 
+#' forecast date
+#' @param truth_source character specifying where the truths will be loaded from.
+#' Currently supports "JHU","USAFacts", "NYTimes". Default to "JHU".
+#' @param truth_as_of the plot includes the truth data that would have been 
+#' in real time as of the truth_as_of date (not using this parameter when truth data 
+#' is from github repo)
+
+#' 
+#' @return ggplot graph
+#' 
+#' @export
+plot_forecast <- function(forecast_data,
+                          model,
+                          target_variable,
+                          location,
+                          intervals = c(.5, .8, .95),
+                          horizon,
+                          truth_source = "JHU",
+                          truth_as_of = NULL){
```

Comment:
This was not in the google doc describing this, but I suggest we add a `plot` argument defaulting to `TRUE`, where at the end of the function we `print(graph)` if `plot` is `TRUE`.